lib/scss_lint/linter/color_variable.rb

```diff
@@ -55,11 +55,13 @@ def literal_string?(script_string)
     def in_variable_declaration?(node)
       parent = node.node_parent
       parent.is_a?(Sass::Script::Tree::Literal) &&
-        parent.node_parent.is_a?(Sass::Tree::VariableNode)
+        (parent.node_parent.is_a?(Sass::Tree::VariableNode) ||
+         parent.node_parent.node_parent.is_a?(Sass::Tree::VariableNode))
```

Comment:
I didn't see any guards elsewhere in this file for `parent_node` being nil. Should we sprinkle in some nil checks?

Reply:
There will always be a parent (at the very least, a `Sass::Tree::RootNode`).